lib/backend.ml

```diff
@@ -381,7 +385,8 @@ let behavior config =
                  let responses =
                    List.map
                      (fun resp ->
-                       Merlin.Response.(strip_file @@ crop_timing @@ resp))
+                       Merlin.Response.(
+                         strip_file @@ crop_arbitrary_keys [ "timing" ] resp))
```

Comment:
Is this the `behavior` cmd? The idea of your PR was to make the `behavior` command deterministic on the `merl-an` side directly, right (as opposed to filtering out the nondeterministic data afterwards)?
```suggestion
                       Merlin.Response.(
                         strip_file @@ crop_arbitrary_keys [ "timing"; "heap_mbytes"; "cache" ] resp))
```

Reply:
You are right, it is fixed on last commit. Thanks